### DISCUSSION TITLE
Rest of Season Projections endpoint / recent change in API?

### DISCUSSION THREAD

**Discussion by TheMathNinja** [cwendt94/espn-api#575] [ESPN API]
Hey guys, wanted to come to the ESPN API experts with a question about locating "rest of season" projections in the ESPN Fantasy API.  This morning when running my usual scrape on ESPN player projections, I noticed that my code for getting weekly projections all of a sudden started returning "Rest of Season" projections (does this strange behavior have anything to do with the [Pull Request](https://github.com/cwendt94/espn-api/pull/572)) this week about weekly projections??). ESPN updated it quickly enough but it made me realize that there were these "rest of season" projections living in the API and I hadn't realized.

I'm using this R package: ffespn. (https://github.com/rlynch91/ffespn/tree/main/R) (you can view all API calls and projections function there).

Currently, a call like ffespn_projections(season = 2024, week = 3, pos = "LB", league_id = my_league_id_2024) will return all week 3 projections for the LB position, and `ffespn_projections(2024, 0, "LB", league_id = my_league_id_2024)` will return preseason yearlong projections. I'd like to be able to add an input to my function called "type" that allows me to see both "this week" and "rest of season" projections for week 3. 

Could you point me to the place in your code that pulls Rest of Season projections by accessing that location in the API or do you not have that? I was wondering if the filters "00" or "10" have something to do that on line 126 of your [espn_requests.py](https://github.com/cwendt94/espn-api/blob/master/espn_api/requests/espn_requests.py) file.

**Comment by TheMathNinja** [comment]
As a quick update on the behavior, I'm now getting a situation where supplying my league ID (1771399141) yields true weekly projections for: QB, WR, K, P, DT, DE, LB

and "Rest of Season" projections for: RB, TE, CB, S.

However, if I drop the league_id parameter, I get weekly projections for: QB, RB, WR (non-rookies), TE, K (non-rookies)

and Rest of Season projections for: Rookie WRs, Rookie Ks

**Comment by cwendt94** [comment]
For this package we get the full season stats when getting all of the league data. In the player stats field `scoringPeriod: 0` represents the full season. And then `statSourceId: 0` represents current amount of points and `1` represents projected amount. This should get you full season stats. 

For the player card api call if you are specifically getting an individual player data then you have to define the additional stat values which will then return full season that is the `00` value.
